### DISCUSSION TITLE
HEVC播放无画面

### DISCUSSION THREAD

**Discussion by mlch911** [Richasy/Bili.Uwp#322] [问答]
使用HEVC播放完全无画面（有弹幕）
使用H.264，打开视频页会卡顿大概3秒，然后才能进入视频页，切换播放速度也会卡住

系统版本号19043.1237
7920x+1080ti

**Comment by Richasy** [comment]
HEVC是要解码器的，你是否安装了？

**Reply by mlch911** [reply]
> HEVC是要解码器的，你是否安装了？

抱歉，装了就好了。不过播放卡顿依旧，播放时快进快退也会卡一两秒
